backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -103,6 +103,15 @@ public PimsAcquisitionFile Add(PimsAcquisitionFile acquisitionFile)
                 }
             }
 
+            // Existing properties should not be added.
+            foreach (var acquisitionPeople in acquisitionFile.PimsAcquisitionFilePeople)
```

Comment:
Existing people?

Reply:
This is add/create file, the flow will be always to add new people. Please let me know if you feel this can be different

Reply:
done. removed